frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -9,11 +9,14 @@
 import com.intellij.ui.SimpleTextAttributes;
 import com.intellij.ui.table.JBTable;
 import com.intellij.util.ui.JBUI;
+import io.vavr.collection.List;
 import org.checkerframework.checker.guieffect.qual.UI;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public abstract class BaseGraphTable extends JBTable {
+  private static List<Integer> rowWidths = List.empty();
```

Comment:
Hmmm shouldn't it be an instance field? What if there is more than `BaseGraphTable` instance in the given JVM (when help/discover action is launched and/or multiple projects are opened)?

Reply:
Whoops, right.